src/main/java/org/kuali/kfs/coa/document/validation/impl/AccountGlobalRule.java

```diff
@@ -1600,63 +1575,49 @@ protected boolean checkCloseAccount(AccountGlobalDetail detail) {
-	  */
-	protected boolean checkAccountExpirationDateValidTodayOrEarlier(CuAccountGlobal newAccount) {
-		// get the expiration date, if any
-		Date expirationDate = newAccount.getAccountExpirationDate();
-		if (ObjectUtils.isNotNull(expirationDate)) {
-
-			Date todaysDate = new Date(getDateTimeService().getCurrentDate().getTime());
-			todaysDate.setTime(DateUtils.truncate(todaysDate,Calendar.DAY_OF_MONTH).getTime());
-
-			// when closing an account, the account expiration date must be the
-			// current date or earlier
-			expirationDate.setTime(DateUtils.truncate(expirationDate, Calendar.DAY_OF_MONTH).getTime());
-			if (expirationDate.after(todaysDate)) {
-				putFieldError(KFSPropertyConstants.ACCOUNT_EXPIRATION_DATE, KFSKeyConstants.ERROR_DOCUMENT_ACCMAINT_ACCT_CANNOT_BE_CLOSED_EXP_DATE_INVALID);
-				return false;
+	 protected boolean checkAccountExpirationDateValidTodayOrEarlier(AccountGlobalDetail detail, CuAccountGlobal newAccount) {
+
+		 //Check user entered account global expiration date first, not required to be entered.
+		 Date userEnteredExpirationDate = newAccount.getAccountExpirationDate();
+		 if (ObjectUtils.isNotNull(userEnteredExpirationDate)) {
+			 //when closing an account, the account expiration date must be the current date or earlier
+			 if (!isExpirationDateCurrentDateOrEarlier(userEnteredExpirationDate)) {
+				 putFieldError(KFSPropertyConstants.ACCOUNT_EXPIRATION_DATE, KFSKeyConstants.ERROR_DOCUMENT_ACCMAINT_ACCT_CANNOT_BE_CLOSED_EXP_DATE_INVALID);
+				 return false;
 			}
-		}
+		 }
+		 else {
+			 //Ensure expiration date already exists on the account being closed
+			 Date accountExistingExpriationDate = detail.getAccount().getAccountExpirationDate();
 
-		return true;
-	}
+			 if (ObjectUtils.isNotNull(accountExistingExpriationDate)) {
+				 if (!isExpirationDateCurrentDateOrEarlier(accountExistingExpriationDate)) {
+					 putFieldError(KFSPropertyConstants.ACCOUNT_EXPIRATION_DATE, CUKFSKeyConstants.ERROR_DOCUMENT_ACCT_GLB_MAINT_CANNOT_BE_CLOSED_EXP_DATE_NOT_ENTERED_EXISTING_DATE_INVALID, new String[]{detail.getAccount().getChartOfAccountsCode(), detail.getAccount().getAccountNumber()});
+					 return false;
+				 }
+			 }
+			 else {
+				 //no expiration date already on the account and none entered on global page, error
+				 putFieldError(KFSPropertyConstants.ACCOUNT_EXPIRATION_DATE, KFSKeyConstants.ERROR_DOCUMENT_ACCMAINT_ACCT_CANNOT_BE_CLOSED_EXP_DATE_INVALID);
+				 return false;
+			 }
+		 }
+		 return true;
+     }
+
+	 private boolean isExpirationDateCurrentDateOrEarlier(Date expirationDateToVerify) {
+		 Date todaysDate = new Date(getDateTimeService().getCurrentDate().getTime());
+		 todaysDate.setTime(DateUtils.truncate(todaysDate,Calendar.DAY_OF_MONTH).getTime());
+    	 expirationDateToVerify.setTime(DateUtils.truncate(expirationDateToVerify, Calendar.DAY_OF_MONTH).getTime());
+    	 
+    	 if (expirationDateToVerify.after(todaysDate)) {
```

Comment:
Indentation seems a bit off in this method. Also, to simplify the code you could return !expirationDateToVerify.after(todaysDate) ?